packages/design-system-docs/src/pages/components/ChoiceList/checkbox-radio.example.html

```diff
@@ -2,58 +2,68 @@
   <fieldset class="ds-c-fieldset">
     <legend class="ds-c-label"><span>Checkbox example</span></legend>
     <span class="ds-c-field__hint">Helpful hint text</span>
-    <input
-      class="ds-c-choice"
-      id="checkbox-1"
-      type="checkbox"
-      name="checkbox-choices"
-      value="checkbox-1"
-      checked
-    />
-    <label for="checkbox-1">
-      <span>Choice 1</span>
-    </label>
-    <input
-      class="ds-c-choice"
-      id="checkbox-2"
-      type="checkbox"
-      name="checkbox-choices"
-      value="checkbox-2"
-    />
-    <label for="checkbox-2">
-      <span>Choice 2</span>
-    </label>
-    <input
-      class="ds-c-choice"
-      id="checkbox-3"
-      type="checkbox"
-      name="checkbox-choices"
-      value="checkbox-3"
-      disabled
-    />
-    <label for="checkbox-3">
-      <span>Disabled choice 3</span>
-    </label>
+    <div>
```

Comment:
Did we add these `divs` just to be consistent with how our `Choice` component renders with a div?

Reply:
OK, I can remove the `<div>` s